apps/web/src/pages/Communities/CommunityLayout.tsx

```diff
@@ -66,7 +66,7 @@ const CommunityLayout = () => {
 
   const roleAssignmentsFiltered =
     data?.myAuth?.roleAssignments?.filter(notEmpty) ?? [];
-  const canAdmin = checkRole(
+  const canAdmin = hasRole(
     [ROLE_NAME.PlatformAdmin, ROLE_NAME.CommunityAdmin],
```

Comment:
Do we have a permission we can use here? Or do we expect to sometimes just pass a one-off array of roles?

Reply:
In some cases, yeah it'd be one-offs, but not here. https://github.com/GCTC-NTGC/gc-digital-talent/pull/12007/commits/8e0598ea3565b544d8f1f0e50221c082e7bc2707